docs/configuration/acme.md

```diff
@@ -165,9 +165,26 @@ storage = "acme.json"
 # ...
 ```
 
-File or key used for certificates storage.
+`storage` allows providing to Træfik an item where storing all the ACME certificates.
 
-**WARNING:** If you use Træfik in Docker, you have 2 options:
+There are two kind of `storage` :
+- a JSON file,
+- a KV store entry.
+
+!!! note
```

Comment:
You may use `!!! danger "DEPRECATED"` instead as in the rest of the documentation for deprecated features.